### ISSUE TITLE
大神方不方便给个联系方式？邮箱也可以 有挺多问题想请教的

### ISSUE THREAD

**Issue by CXXXXv** [Byronnar/tensorflow-serving-yolov3#5]

**Comment by Byronnar** [comment]
你好，请问有什么问题，可以直接提出

**Comment by i-chaochen** [comment]
感谢分享这么好的代码
@Byronnar    
请问为什么compute_loss 里为什么要对label 和 true 分别都要用到 large，medium 和small 3个bbox？难道ground true不就一个bbox？  我这个有点看不太明白，希望解答，谢谢！

**Comment by i-chaochen** [comment]
> 感谢分享这么好的代码
> @Byronnar
> 请问为什么compute_loss 里为什么要对label 和 true 分别都要用到 large，medium 和small 3个bbox？难道ground true不就一个bbox？（也就是 true 就一个 bbox） 我这个有点看不太明白，希望解答，谢谢！

哈，自己reply自己一下，不知道理解的对不对。希望指正

因为yolo的loss function是对bbox的一个loss，所以相对而言这个ground true也是一个指定anchor box的一个loss，所以也就有了相应的3个大中小的3个loss

**Comment by Byronnar** [comment]
嗯,也可以这么理解,但是 主要是因为 config文件里面设置了__C.YOLO.STRIDES                = [8, 16, 32],在三个不同的尺度上计算舒适函数更加准确

**Comment by i-chaochen** [comment]
> 嗯,也可以这么理解,但是 主要是因为 config文件里面设置了__C.YOLO.STRIDES = [8, 16, 32],在三个不同的尺度上计算舒适函数更加准确

借道再顺便问个

loss function = loss of bbox (giou_loss + conf_loss) + loss + obj_classification  (prob_loss)

比如这个prob_loss  是 log loss（sigmoid_cross_entropy_with_logits）算出来的

为啥这些loss function 都不加 regularization？

**Comment by i-chaochen** [comment]
> 嗯,也可以这么理解,但是 主要是因为 config文件里面设置了__C.YOLO.STRIDES = [8, 16, 32],在三个不同的尺度上计算舒适函数更加准确

借道再顺便问个问题希望大神能解答一下

loss function = loss of bbox (giou_loss + conf_loss) + loss + obj_classification  (prob_loss)

比如这个prob_loss  是 log loss（sigmoid_cross_entropy_with_logits）算出来的

为啥这些loss function 都不加 regularization？

**Comment by Byronnar** [comment]
这个你可以自己尝试加一下
